### PR TITLE
Add missing alt text to images

### PR DESCRIPTION
**Pull request by fanyak**
#### Summary

Add missing alt texts for images catalogued [here](https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0)

#### Motivation

The changes are meant to improve accessibility.  The content of images that don't have alt text is inaccessible.

#### Supporting details

Changes where motivated by the discussion [here](https://github.com/orgs/mdn/discussions/168#discussioncomment-3361122)

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/techniques/3d_collision_detection/bounding_volume_collision_detection_with_three.js/index.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ We have prepared some [live demos](https://mozdevs.github.io/gamedev-js-3d-aabb/
 - [Box vs. Box and Sphere](https://mozdevs.github.io/gamedev-js-3d-aabb/raw_box.html)
 - [Sphere vs. Box and Sphere](https://mozdevs.github.io/gamedev-js-3d-aabb/raw_sphere.html)
 
-![](screen_shot_2015-10-20_at_15.19.16.png)
+![A Knot object, a large Sphere object and a small Sphere object in 3-D space. Three vectors are drawn on the small Sphere. The vectors point in the directions of the three axes that define the space. Text at the bottom reads: Drag the ball around.](screen_shot_2015-10-20_at_15.19.16.png)
```
Comment (Rumyra) [inline files/en-us/games/techniques/3d_collision_detection/bounding_volume_collision_detection_with_three.js/index.md:128]:
```suggestion
![A knot object, a large sphere object and a small sphere object in 3-D space. Three vectors are drawn on the small sphere. The vectors point in the directions of the three axes that define the space. Text at the bottom reads: Drag the ball around.](screen_shot_2015-10-20_at_15.19.16.png)
```

File: files/en-us/games/techniques/3d_collision_detection/bounding_volume_collision_detection_with_three.js/index.md [hunk 2]
```diff
@@ -182,4 +182,4 @@ box3.containsPoint(point.position);
 
 There are **two demos** you can take a look at on our [live demos page](https://mozdevs.github.io/gamedev-js-3d-aabb/). The [first one](https://mozdevs.github.io/gamedev-js-3d-aabb/api_point.html) showcases point vs. box collisions using `BoxHelper`. The [second one](https://mozdevs.github.io/gamedev-js-3d-aabb/api_box.html) performs box vs. box tests.
 
-![](screen_shot_2015-10-19_at_12.10.06.png)
+![A Knot object, a Sphere object and a Cube object in 3-D space. The Knot and the Sphere are encompassed by a virtual bounding box. The Cube is intersecting the bounding box of the Sphere. Text at the bottom reads: Drag the cube around. Press Esc to toggle B-Boxes.](screen_shot_2015-10-19_at_12.10.06.png)
```
Comment (Rumyra) [inline files/en-us/games/techniques/3d_collision_detection/bounding_volume_collision_detection_with_three.js/index.md:185]:
```suggestion
![A knot object, a sphere object and a cube object in 3-D space. The knot and the sphere are encompassed by a virtual bounding box. The cube is intersecting the bounding box of the sphere. Text at the bottom reads: Drag the cube around. Press Esc to toggle B-Boxes.](screen_shot_2015-10-19_at_12.10.06.png)
```

**Review by Rumyra — CHANGES_REQUESTED** [review]

File: files/en-us/games/techniques/3d_collision_detection/bounding_volume_collision_detection_with_three.js/index.md (hunk 1 above)
Comment (Rumyra) [reply files/en-us/games/techniques/3d_collision_detection/bounding_volume_collision_detection_with_three.js/index.md:128]: These look great @fanyak we just don't need to capitalise the objects :)

**Comment by fanyak** [comment]
@Rumyra thank you for the help!

**Review by estelle — APPROVED** [review]
Awesome. Thanks!

**Review by Rumyra — APPROVED** [review]

**Pull request merged**